src/pip-delete-this-directory.txt

```diff
@@ -0,0 +1,5 @@
+This file is placed here by pip to indicate the source was put
```

Comment:
Pls be careful with `git add -A` or `git commit -A` -- the `src` dir shouldn't be included.